packages/builders/scripts/testing/recorded-retired-instances.js

```diff
@@ -36,6 +36,12 @@ export const testRecordedRetiredInstances = async ({
   const committeeInstance = retiredContractInstances.get(committeeIDs[0]);
   assert(await E(contractKits).get(committeeInstance));
 
+  const charterIDs = Array.from(retiredContractInstances.keys()).filter(k =>
```

Comment:
I'm used to seeing...

```suggestion
  const charterIDs = [...retiredContractInstances.keys()].filter(k =>
```

Reply:
done